aiohttp/streams.py

```diff
@@ -122,7 +122,7 @@ def __init__(
         self._waiter: Optional[asyncio.Future[None]] = None
         self._eof_waiter: Optional[asyncio.Future[None]] = None
         self._exception: Optional[BaseException] = None
-        self._timer = timer
+        self._timer = timer if timer else TimerNoop()
```

Comment:
Updated this for simplicity and consistency, as mypy required me to update some related parts anyway.